chromium/.eslintrc.json

```diff
@@ -1,16 +1,16 @@
 {
   "env": {
     "browser": true,
-    "commonjs": true,
     "es6": true,
-    "node": true,
```

Comment:
Was node supported at some point?

Reply:
https://github.com/EFForg/https-everywhere/blob/master/rewriter/rewriter.js

Reply:
I removed `commonjs` and `node` from `env` list to make `exported` and `global` work.

Reply:
Could you add `/* eslint-env node, commonjs, etc  */` to the files where it is appropriate?

Reply:
No, it would make ESLint ignore `/* exported */` directives.

Reply:
Sorry I'm not very familiar with eslint. Doesn't this file only apply this configuration  to sub-directories of `chromium/`? Since `rewriter.js` isn't in this directory, and also doesn't have any exported directives, why can't we add `/* eslint-env node */` to it?

Reply:
I only changed files under `chromium` in this PR.